sos/report/plugins/__init__.py

```diff
@@ -1569,6 +1569,21 @@ def add_blockdev_cmd(self, cmds, devices='block', timeout=300,
                              whitelist=whitelist, blacklist=blacklist,
                              tags=_dev_tags)
 
+    def add_tape_cmd(self, cmds, devices='tape', timeout=300,
+                         sizelimit=None, chroot=True, runat=None, env=None,
+                         binary=False, prepend_path=None, whitelist=[],
+                         blacklist=[], tags=[]):
+        _dev_tags = []
+        prepend_path = prepend_path or '/dev/'
+        devices = self.devices['tape']
+        _dev_tags.append('block')
```

Comment:
Tape devices are normally character devices not block devices.

Reply:
I missed this one when copying from the other block, sorry!